### PR TITLE
Lesson16

### PR DESCRIPTION
**Pull request by carlotrimarchi**

### CODE REVIEW

**Comment by carlotrimarchi** [comment]
That's most of the content, but tomorrow (Wednesday) I will add a recap at the end and some exercise to do in class and something for the weekend.

File: lesson16.md [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+Lesson 16, Tuesday, 2020-10-20
+
+---
+
+### The DOM API
+
+1. what's the DOM API?
+  - it's an interface that lets us interact with our web page
+  <!-- .element: class="fragment" -->
+1. what can we do with?
```
Comment (sevtapim) [inline lesson16.md:14]:
```suggestion
1. what can we do with it?
```

File: lesson16.md [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+  - we can access HTML elements
+  <!-- .element: class="fragment" -->
+  - we can read from HTML elements
+  <!-- .element: class="fragment" -->
+  - we can change HTML elements content
```
Comment (sevtapim) [inline lesson16.md:19]:
```suggestion
  - we can change HTML elements' content
```

File: lesson16.md [hunk 3]
```diff
@@ -0,0 +1,340 @@
...
+document.body.style.color = "white";
+document.body.style.fontSize = "48px";
+document.body.textContent = "Hello!";
+```
+
+
+---
+
+### The DOM API: examples
+Clear the content of a `div`, a `paragraph` and an `input`:
```
Comment (sevtapim) [inline lesson16.md:106]:
```suggestion
Clear the contents of a `div`, a `paragraph` and an `input`:
```

File: lesson16.md [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+myInputElement.value = '';
+```
+<!-- .element: class="fragment" -->
+
+---
+
+
+### The DOM API
+
+What's the difference between those element: `h1`, `h2`, `p`, `div`
```
Comment (sevtapim) [inline lesson16.md:130]:
```suggestion
What's the difference between those elements: `h1`, `h2`, `p`, `div`
```

File: lesson16.md [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+<div><a href="http://example.com">A link</a> inside a div.</div>
+```
+
+and those: `input`, `img`
+```html
+<input type="text" id="color" value="red">
+<img src="myImage.jpg">
+```
+
+The content of `input` and `img` is controlled by an element `attribute`.
```
Comment (sevtapim) [inline lesson16.md:145]:
```suggestion
The contents of `input` and `img` are controlled by an element' `attribute`.
```

**Pull request merged**